pkg/set/set_bench_test.go

```diff
@@ -0,0 +1,31 @@
+package set
+
+import (
+	"strconv"
+	"testing"
+)
+
+var (
+	a IntSet
+	s StringSet
+)
```

Comment:
Why are these global vars, and not just local variables defined in the `Benchmark..` function via `:=`?

Reply:
As recommended by the link you suggested, these global vars may help to ensure the benchmark code is not optimized out.

Reply:
SG